pytripgui/canvas_vc/bars/ctx_bar.py

```diff
@@ -0,0 +1,19 @@
+from matplotlib.projections import register_projection
+
+from pytripgui.canvas_vc.bars.bar import Bar
+from pytripgui.canvas_vc.bars.projection_enum import BarProjection
+
+
+# define new class that inherits from Axes
+# class attribute - name - is very important

```

Comment:
the comment is useless - it doesn't explain WHY the `name` is importnat

Reply:
It was some kind of prelude to next comment:
![image](https://user-images.githubusercontent.com/44035398/126372638-7bcce862-258e-4b05-8ffc-33427aead248.png)